lib/cc/analyzer/engines_runner.rb

```diff
@@ -72,12 +75,16 @@ def metadata(engine_name)
         @registry[engine_name]
       end
 
-      def exclude_paths
-        PathPatterns.new(@config.exclude_paths || []).expanded + gitignore_paths
+      def include_paths
+        if paths && paths.any?
```

Comment:
If `paths` is any array, can we get away with just doing `if paths.present?` instead?